docs/shared-content/tentacle/tentacle-downloads.include.md

```diff
@@ -1,19 +1,11 @@
 ## Download the Tentacle installer
 
-### Long-term support Octopus Server (Slow lane)
-
-The version of Tentacle that works with the latest long-term support version of Octopus Deploy:
-
-- [Octopus Tentacle x64](https://octopus.com/downloads/slowlane/WindowsX64/OctopusTentacle)
-- [Octopus Tentacle 32-bit/x86](https://octopus.com/downloads/slowlane/Windows/X86WindowsX86/OctopusTentacle)
+- [Octopus Tentacle](https://octopus.com/downloads/slowlane/WindowsX64/OctopusTentacle)
 - [Octopus Tentacle for Linux](https://octopus.com/downloads/slowlane/Linux_x64TarGz/OctopusTentacle)
 
-If you need an older version of Tentacle that works with a previous long-term support release, visit the [previous downloads page](https://octopus.com/downloads/previous), locate the specific version of Octopus you're working with, and click your version to visit the downloads page for that version.
-
-### Fast lane Tentacle
-
-The version of Tentacle that works with the latest fast lane version of Octopus Deploy:
+:::warning
+If you are using Windows Server 2003 with .NET Framework 4.0, you will need to download either:
+- [Octopus Tentacle x64](https://download.octopusdeploy.com/octopus/Octopus.Tentacle.3.0.26.0-x64.msi)
+- [Octopus Tentacle 32-bit/x86](https://download.octopusdeploy.com/octopus/Octopus.Tentacle.3.0.26.0.msi)
 
- - [Octopus Tentacle x64](https://octopus.com/downloads/fastlane/WindowsX64/OctopusTentacle)
- - [Octopus Tentacle 32-bit/x86](https://octopus.com/downloads/fastlane/WindowsX86/OctopusTentacle)
- - [Octopus Tentacle for Linux](https://octopus.com/downloads/fastlane/Linux_x64TarGz/OctopusTentacle)
+If you need an older version of Tentacle that works with a previous long-term support release, visit the [previous downloads page](https://octopus.com/downloads/previous), locate the specific version of Octopus you're working with, and click your version to visit the downloads page for that version.
```

Comment:
We shouldn't advise this. The latest version of Tentacle should be highly compatible. I think the warning only applies to Windows Server 2003.

Reply:
Removed